src/components/text-input/index.md.njk

```diff
@@ -61,17 +61,29 @@ Fluid width inputs will resize with the viewport.
 
 ### Numbers
 
-If it’s likely that the user will need to enter a number and you want to bring up the numeric keypad on a mobile device, set the `inputmode` attribute to `numeric` and the `pattern` attribute to `[0-9]*`. See how to do this in the HTML and Nunjucks tabs in the following example.
+#### Asking for whole numbers
 
-{{ example({group: "components", item: "text-input", example: "number-input", html: true, nunjucks: true, open: false, size: "m"}) }}
+If you're asking the user to enter a whole number and you want to bring up the numeric keypad on a mobile device, set the `inputmode` attribute to `numeric` and the `pattern` attribute to `[0-9]*`. See how to do this in the HTML and Nunjucks tabs in the following example.
 
-Do not use `<input type=”number”>` unless your user research shows that there’s a need for it. With `<input type=”number”>` there’s a risk of users accidentally incrementing a number when they’re trying to do something else - for example, scroll up or down the page. And if the user tries to enter something that’s not a number, there’s no explicit feedback about what they’re doing wrong.
+{{ example({group: "components", item: "text-input", example: "number-input", html: true, nunjucks: true, open: false, size: "m"}) }}
 
 There are specific patterns for:
 
 - [dates](/patterns/dates/)
 - [telephone numbers](/patterns/telephone-numbers/)
 
+#### Asking for numbers with decimal places
+
+If you're asking the user to enter a number that might include a decimal place, use `input type="text"` without `inputmode` or `pattern` attributes.
+
+Do not set the `inputmode` attribute to `decimal` as it causes some devices to bring up a keypad without a period key, preventing users from entering a decimal point.
```

Comment:
> without a period key

I think the term ['decimal point key'](https://en.wikipedia.org/wiki/Numeric_keypad#:~:text=(decimal%20point)) is the correct one.